server/models/SpotifyToken.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const SpotifyTokenSchema = new mongoose.Schema({
+  userId: String,
+  spotifyId: String,
+  accessToken: String,
+  refreshToken: String,
+  expiresIn: Number,
+  timeStamp: Number
```

Comment:
This should be a date. Mongo has a date type and mongoose gives you access to it:
https://mongoosejs.com/docs/schematypes.html
https://www.tutorialspoint.com/mongodb/mongodb_datatype.htm